### PR TITLE
build: prepush should try running tests since master revision

### PR DESCRIPTION
**Pull request by priyajeet**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: scripts/prepush.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ prepush() {
     printf "${blue}-------------------------------------------------------------${end}"
     printf "${blue}Testing${end}"
     printf "${blue}-------------------------------------------------------------${end}"
-    yarn test --lastCommit || exit 1
+    yarn test --changedSince=master || exit 1
```
Comment (jstoffan) [inline scripts/prepush.sh]: Jest is so good.

Comment (priyajeet) [reply scripts/prepush.sh]: unsure tho this is correct

File: scripts/prepush.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ prepush() {
     printf "${blue}-------------------------------------------------------------${end}"
     printf "${blue}Testing${end}"
     printf "${blue}-------------------------------------------------------------${end}"
-    yarn test --lastCommit || exit 1
+    yarn test --changedSince=upstream/master || exit 1
```
Comment (j3tan) [inline scripts/prepush.sh]: add a `git fetch upstream` above this

**Pull request merged**